### PR TITLE
Update README with certificate section

### PR DESCRIPTION
**Pull request by jekuaitk**
#### Link to ticket

https://leantime.itkdev.dk/#/tickets/showTicket/510

#### Description

Updates `README.md` with a certificate section.

#### Screenshot of the result

N/A

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,30 @@ vendor/bin/drush pm:enable os2forms_fasit
...
+```
+
+Example output
+
+```sh
+SHA1 Fingerprint=6A:CB:26:1F:39:31:72:D8:7F:A3:99:7C:EC:86:56:97:59:A8:52:8A
+```
+
+which should be converted to lowercase and have its colons removed,
+before provided to Fasit Schultz, i.e.
```
Comment (rimi-itk) [inline README.md]:
```suggestion
which should be converted to lowercase and have its colons removed before provided to Fasit Schultz, i.e.
```

File: README.md [hunk 2]
```diff
@@ -15,6 +15,30 @@ vendor/bin/drush pm:enable os2forms_fasit
...
+openssl x509 -in SOME_CERTIFICATE.pem -noout -fingerprint
+```
+
+Example output
+
+```sh
+SHA1 Fingerprint=6A:CB:26:1F:39:31:72:D8:7F:A3:99:7C:EC:86:56:97:59:A8:52:8A
+```
+
+which should be converted to lowercase and have its colons removed,
```
Comment (rimi-itk) [inline README.md]: The command line is your friend:

```shell
openssl x509 -in SOME_CERTIFICATE.pem -noout -fingerprint |  cut -d= -f2 | sed 's/://g' | tr '[:upper:]' '[:lower:]'
```

(that's just one way to do it)

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jekuaitk) [reply README.md]: Updated with command.

File: README.md [hunk 3]
```diff
@@ -15,6 +15,23 @@ vendor/bin/drush pm:enable os2forms_fasit
 Configure Fasit API `base url` and a way of getting
 certificate on `/admin/os2forms_fasit/settings`.
 
+### Certificate
+
+The certificate must be in `pem` or `cer` format and
+must be whitelisted by Fasit Schultz.
+For this the certificate thumbprint is needed. To get the thumbprint,
+in the needed format, from the command line run
```
Comment (rimi-itk) [inline README.md]: Previously, you told what “the needed format” is. I think you should do that (again) since it's important information.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

**Review by rimi-itk — APPROVED** [review]
🥇

**Pull request merged**